src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts

```diff
@@ -14,7 +15,7 @@ import getMatchingBottomTabRouteForState from './getMatchingBottomTabRouteForSta
 import getMatchingCentralPaneRouteForState from './getMatchingCentralPaneRouteForState';
 import replacePathInNestedState from './replacePathInNestedState';
 
-const RHP_SCREENS_OPENED_FROM_LHN = [SCREENS.SETTINGS.SHARE_CODE, SCREENS.SETTINGS.PROFILE.STATUS] as const;
+const RHP_SCREENS_OPENED_FROM_LHN: Screen[] = [SCREENS.SETTINGS.SHARE_CODE, SCREENS.SETTINGS.PROFILE.STATUS, SCREENS.SETTINGS.PREFERENCES.PRIORITY_MODE];
```

Comment:
Why has `as const` been removed?

Reply:
I wanted to narrow down the typing, as currently, we don't enforce the list items to be of `Screen` type. I think we can go with the following:
```javascript
const RHP_SCREENS_OPENED_FROM_LHN = [...] satisfies Screen[];
``` 

In this case we:
- enforce the list items to be of `Screen` type
- narrow down the typing (similarly to `as const`).

If this approach doesn't work, I can switch back to the original one.

WDYT?

Reply:
Sounds fine, thanks for the explanation :)